### DISCUSSION TITLE
I can not see DEV branch. Only main.

### DISCUSSION THREAD

**Discussion by nicvagn** [HarshilPatel007/YACS#5] [General]
I can not see the dev branch. Only main. 

I am new to working with other peoples remotes.

**Comment by HarshilPatel007** [comment]
@nicvagn ,
`git checkout dev`

**Reply by nicvagn** [reply]
git checkout dev
branch 'dev' set up to track 'origin/dev'.
Switched to a new branch 'dev'

**Reply by HarshilPatel007** [reply]
@nicvagn ,

- first delete this cloned repo
- clone this repo again `git clone https://github.com/HarshilPatel007/YACS`
- then `git branch --list -a`
- you'll be able to see the all other local/remote branches
- you can then select which branch to work with
- `git checkout dev` it'll create a local `dev` branch from the remote `dev` branch
- to confirm `git log --oneline` and check the latest commits on `dev` branch. compare it.


or just setup `zsh` => https://ohmyz.sh & install a git plugin it'll show you the current branch you're working in
![image](https://github.com/HarshilPatel007/YACS/assets/16424866/715832db-d3e1-41eb-a9d3-b5d3d3e011d6)

**Reply by HarshilPatel007** [reply]
> git checkout dev
> branch 'dev' set up to track 'origin/dev'.
> Switched to a new branch 'dev'
> 
> 

That's it.
Can you please post a screenshot of it?

**Reply by nicvagn** [reply]
I tried that, and did as you say. Still same prob

**Reply by nicvagn** [reply]
of what? sorry

**Reply by nicvagn** [reply]
I am sorry for my ignorance. Thanks for your graciousness.

**Reply by HarshilPatel007** [reply]
@nicvagn , no problem.
You can use zsh. See above comment. It'll make your life a lot better.

**Reply by nicvagn** [reply]
I installed https://github.com/ohmybash/oh-my-bash

I am scared of the letter z

**Reply by HarshilPatel007** [reply]
@nicvagn , 😂 typical Canadian..!!!

**Reply by nicvagn** [reply]
Sorry

**Comment by nicvagn** [comment]
![image](https://github.com/HarshilPatel007/YACS/assets/37600758/0ff04167-f441-497f-8191-ee46558af469)

**Reply by HarshilPatel007** [reply]
@nicvagn , that's it. you're on a dev branch.
please see this https://github.com/HarshilPatel007/YACS/discussions/5#discussioncomment-8608999

**Comment by nicvagn** [comment]
I was missing the --all in git branch --all. I am sorry for the confusion.

**Comment by nicvagn** [comment]
git fetch origin

is what I could not grasp.
